admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -1065,3 +1065,63 @@
     }
   }
 }
+
+#select2-product_combination_bulk_images-results .select2-results__option--highlighted[aria-selected],
+#select2-product_combination_bulk_images-results .select2-results__option[aria-selected=true] {
+  background-color: inherit;
+  color: inherit;
+}
+
+.combination-bulk-images {
+  width: 2.857rem;
+  height: 2.857rem;
+  background-color: #bbcdd2;
+  border: solid 0.07143rem #bbcdd2;
+}
+
+.combination-bulk-images-label {
+  height: 1.429rem;
+  object-fit: contain;
+  font-family: $font-family-base;
+  font-size: 0.9rem;
+  line-height: 1.43;
+  color: $gray-medium;
+  margin: 0.6429rem 0 0 0.6429rem;
+}
+
+#select2-product_combination_bulk_images-results li.select2-results__option input[type=checkbox] {
+  margin: 0 0.625rem 0 0.313rem;
+  vertical-align: middle;
+}
+
+#select2-product_combination_bulk_images-results::-webkit-scrollbar {
```

Comment:
Why are you using the browser prefixes? It is automatically added with webpack...

Reply:
I think that this webpack autofixer version does not handle this for scrollbar selector. 
https://github.com/postcss/autoprefixer/issues/821